default.nix

```diff
@@ -15,6 +15,12 @@ let
             text
             hlint
             p.ihp
+            tagsoup
+            containers
+            network-uri
+            utf8-string
```

Comment:
Why are these needed?

Reply:
these three libraries where being used in custom implementation of html sanitizer , but now been removed.